README.md

```diff
@@ -2,19 +2,34 @@
 A simple program to remove the watermark from a PDF file. 
 
 
-### how?
+### How?
 
 1. convert the PDF file into images using `pdf2image`
 2. convert the images to numpy array
 3. find the specific pixel by watermarks' rgb values and change them into (255,255,255)
 4. save the modified images
 
 
-### environment
-`pdf2image`: pip install pdf2image
+### Environment
 
-`poppler`: brew install poppler
+First you need to install the dependencies:
+```
+$ pip install pdf2image
+```
+```
+$ pip install scikit-image
+```
 
+Inside the repository create a directory that will receive the modified images:
+```
+$ mkdir jiangyi3
+```
+To execute:
+```
+$ python watermark.py
+```
+Don't forget to indicate the pdf's path you want to convert.
 
-### results
+
+### Results
```

Comment:
What if I want to read/load only image and remove watermark instead of pdf file?